TASVideos.WikiEngine/NodeImplementations/Element.cs

```diff
@@ -164,6 +165,18 @@ public async Task WriteTextAsync(TextWriter writer, WriterContext ctx)
 		}
 	}
 
+	public async Task WriteMetaDescriptionAsync(StringBuilder sb, WriterContext ctx)
+	{
+		// write all except for divs that aren't p (which aims to exclude TOC, tabs, etc.)
+		if (Tag != "div" || Attributes.TryGetValue("class", out string? classes) && classes.Split(' ').Contains("p"))
+		{
+			foreach (var c in Children)
```

Comment:
We should break this loop early if the `sb` is full.  If a page has tabs, large portions of its content may be nested inside individual tags and we could save time by stopping the iteration here.